crates/cxx-qt-gen/src/parser/property.rs

```diff
@@ -110,20 +121,36 @@ impl ParsedQProperty {
                 let mut write = None;
                 let mut notify = None;
 
+                let mut constant = false;
+                let mut required = false;
+                let mut reset = None;
+
                 // Create mutable closure to capture the variables for setting with the Meta values
                 let mut update_fields = |ident: &Ident, value: Option<Ident>| -> Result<()> {
-                    let variable = match ident.to_string().as_str() {
-                        "read" => &mut read,
-                        "write" => &mut write,
-                        "notify" => &mut notify,
+                    match ident.to_string().as_str() {
+                        "read" => read = Some(value.map_or(FlagState::Auto, FlagState::Custom)), // Might be able to use DRY here
+                        "write" => write = Some(value.map_or(FlagState::Auto, FlagState::Custom)),
+                        "notify" => notify = Some(value.map_or(FlagState::Auto, FlagState::Custom)),
```

Comment:
did the closure route not work? eg

```
"read" => map_auto_or_custom(&mut read),
```

could even pass in `&value` if capturing is an issue. But then at least you have a closure that does the
`*mutable = Some(value.map_or(FlagState::Auto, FlagState::Custom));`

Reply:
I just forgot to clean that bit up actually, will do

Reply:
done